backend/tracim_backend/tests/functional/test_reset_password.py

```diff
@@ -76,6 +125,48 @@ def test_api__reset_password_request__ok__nominal_case(self):
         assert 'code' in res.json.keys()
         assert res.json_body['code'] == error.NOTIFICATION_DISABLED_CANT_RESET_PASSWORD  # nopep8
 
+    @pytest.mark.unknown_auth
```

Comment:
I think this test case is enough... it's almost a duplicate of functionnal test case, doesnt it ? You do not need to implement the two of them every time.

Reply:
There is 3 different endpoints for email reset and we should be sure that unknown account can use all of them, so i'm not sure if that a good idea to not set those tests, best solution to avoid code duplication in this case can be using parameter : https://docs.pytest.org/en/latest/parametrize.html but to use this, we should probably reconvert test to real pytest first (there are unittest-like actually)